koku/sources/kafka_message_processor.py

```diff
@@ -152,6 +156,11 @@ def save_credentials(self):
             LOG.info(f"[save_credentials] source_type not found for source_id: {self.source_id}")
             return
 
+        if source_type == Provider.PROVIDER_OCI or source_type == Provider.PROVIDER_OCI_LOCAL:
+            # OCI sources do not have authentication, so skip running thru this function
+            LOG.info("[save_credentials] skipping for OCI source")
+            return {}
```

Comment:
**question:** This method already has two return types `None` and `bool` Is it your intention to add a _third_ return type of `dict`?